routes/server.go

```diff
@@ -950,6 +951,13 @@ func (fes *APIServer) NewRouter() *muxtrace.Router {
 			fes.GetHodlersForPublicKey,
 			PublicAccess,
 		},
+		{
+			"GetHoldersForPublicKeyWithLockedBalances",
+			[]string{"POST", "OPTIONS"},
```

Comment:
From the looks of it, this could be a "GET" request. Any reason why you want a POST instead?

Reply:
It prevents us from getting the benefits of caching. Cloudflare can help with caching GET requests.